themes/bootstrap3/js/lightbox.js

```diff
@@ -327,7 +327,7 @@ VuFind.register('lightbox', function Lightbox() {
     _lightboxTitle = submit.data('lightbox-title') || $(form).data('lightbox-title') || false;
     // Get Lightbox content
     ajax({
-      url: $(form).attr('action') || _currentUrl,
+      url: $(form).attr('action') || _currentUrl || window.location.href,
```

Comment:
Just curious what this change is for...

Reply:
I forgot to mention: Lightbox didn't work on forms that don't have an action attribute (the hold selection form is such one). _currentUrl is not yet set at the moment the initial request is made, so we need to use the current page url like the form would have used if submitted normally.

Reply:
Thanks, that makes sense. I wonder if we should split off this one-liner into its own pull request for separate testing/merging. (If you think it's worth it but don't have the time to set it all up, I'm willing to do the work).

Reply:
That'd make sense, and I'd appreciate help on this.

Reply:
Okay, extracted to #1959.

Reply:
I merged #1959 to dev and then merged dev back to here, so these changes are no longer part of the diffs in this PR. Thanks!

Reply:
Thanks!